manifests/params.pp

```diff
@@ -35,5 +44,9 @@
   Stdlib::HTTPSUrl $candlepin_url = "https://${candlepin_host}:${candlepin_port}/candlepin",
   String[1] $candlepin_client_keypair_group = 'foreman',
   String[1] $postgresql_evr_package = $katello::globals::postgresql_evr_package,
+  Boolean $enable_yum = $katello::globals::enable_yum,
+  Boolean $enable_file = $katello::globals::enable_file,
+  Boolean $enable_docker = $katello::globals::enable_docker,
+  Boolean $enable_deb = $katello::globals::enable_deb,
```

Comment:
Do we even need these or can we just use the globals directly in the installer? We probably need to accept `Enum['']` there as well since we use `alias()`.